website_sale_product_detail_attribute_value_image/templates/shop_product.xml

```diff
@@ -28,10 +28,7 @@
                 <t t-foreach="variant_id.value_ids.filtered(lambda x: x.website_product_detail_image_published)" t-as="attr_value">
                     <div class="flex-fill px-1">
                         <div class="text-center">
-                            <span
-                                t-field="attr_value.website_product_detail_image"
-                                t-options="{'widget': 'image', 'resize': '40x40'}"
-                            />
+                            <img class="img" height="40" t-attf-src="/web/image?model=product.attribute.value&amp;field=website_product_detail_image&amp;id=#{attr_value.id}" t-att-title="attr_value.website_name or attr_value.name" t-att-alt="attr_value.website_name or attr_value.name"/>
```

Comment:
why not max-height and max-width?

Reply:
You don't want the resize behaviour?

Reply:
Resize? The problem I see with this is that an image of 1200x40 will break the layout.

Reply:
If only sets the "height" or "width" the image preserve the aspect ratio... but yeah, maybe better force the 40x40px to work as before.

Reply:
That's why I told about max-widht:40 and max-height:40. With that, it adapts to the highest one, and the other is resized in browser preserving aspect ratio.

Reply:
I saw that if the same value is used "max-width" wins... so only need use "max-width" in this case. Thanks!